key-manager/src/main/scala/org/bitcoins/keymanager/KeyManager.scala

```diff
@@ -0,0 +1,136 @@
+package org.bitcoins.keymanager
+
+import java.nio.file.Files
+
+import org.bitcoins.core.compat.{CompatEither, CompatLeft, CompatRight}
+import org.bitcoins.core.crypto._
+import org.bitcoins.core.hd.{HDAccount, HDPath}
+import org.bitcoins.core.util.BitcoinSLogger
+import org.bitcoins.keymanager.util.HDUtil
+import scodec.bits.BitVector
+
+import scala.util.{Failure, Success, Try}
+
+case class KeyManager(
+    private val mnemonic: MnemonicCode,
+    kmParams: KeyManagerParams) {
+
+  private val seed = BIP39Seed.fromMnemonic(mnemonic, BIP39Seed.EMPTY_PASSWORD) // todo think more about this
+
+  private val privVersion: ExtKeyPrivVersion =
+    HDUtil.getXprivVersion(kmParams.purpose, kmParams.network)
+
+  private val rootExtPrivKey = seed.toExtPrivateKey(privVersion)
+
+  /** Converts a non-sensitive DB representation of a UTXO into
+    * a signable (and sensitive) real-world UTXO
+    */
+  def toSign(privKeyPath: HDPath): Sign = {
```

Comment:
Since #959 has been merged, can't this method just return `xpriv` as an `ExtSign`?